index.js

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env node
-const readPkgUp = require("read-pkg-up");
-(async () => {
-  const package = (await readPkgUp()).pkg;
-  Object.entries(package.scripts).forEach(([key, script]) => {
-    console.log(`${key} => ${script}`)
-  })
-})();
+const readPkgUp = require('read-pkg-up');
+const pkgUp = require('pkg-up');
+const packageInfo = readPkgUp.sync();
+
+const package = packageInfo.pkg;
+
+if (
+  typeof package.scripts !== 'object' ||
```

Comment:
probably rare, but scripts might be null so should handle that
```suggestion
  !package.scripts ||
  typeof package.scripts !== 'object' ||
```